Tests/SwiftParserTest/DeclarationTests.swift

```diff
@@ -737,22 +745,26 @@ final class DeclarationTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected async specifier; did you mean 'async'?")
+        DiagnosticSpec(message: "expected async specifier; did you mean 'async'?", fixIts: ["replace 'reasync' with 'async'"])
       ]
     )
   }
 
   func testExpressionMember() {
     assertParse(
       """
-      struct S {1️⃣
-        /2️⃣ ###line 25 "line-directive.swift"3️⃣
+      struct S ℹ️{1️⃣
+        🔟/2️⃣ ###line 25 "line-directive.swift"3️⃣
```

Comment:
Before I continue @ahoppen I got into an issue here. 

We normally use ℹ️ for Note  locationmarkers.
But in this case here we have multiple locations. 

I saw that you in #1550 add line and column. 
1. We could start to use same approach here, then we never hit this again. 
The bad, as it makes it hard to see when we receive things like 2:37.
2. We can also just use another number but then we mix with where we expect the diagnosis.

Reply:
> I saw that you in https://github.com/apple/swift-syntax/pull/1550 add line and column.

My thinking here was that this is for an assertion function that authors of macros (outside of the swift-syntax repository) would use. Since I think the location marker concept has a non-trivial learning curve, I resorted to just using line/columns which everybody understands. For contributors of swift-syntax, I think this learning curve is acceptable because it *significantly* improves our tests.

---

I think there are already some test cases that contain multiple notes and we just the normal `1️⃣` etc. markers for those. I think that’s fine. 

Also wherever possible, I think tests should be split so they ideally only emit a single note. But that’s not how all our existing tests are structured (mostly because the my migration script wasn’t able to split them to a finer-grained level) and it’s not always possible.